src/NUnitTestAdapter/AdapterSettings.cs

```diff
@@ -136,6 +142,16 @@ public void Load(string settingsXml)
             // Force Verbosity to 1 under Debug
             Verbosity = 1;
 #endif
+
+            // If any in proc data collector will be instantiated by the TestPlatform run tests sequentially.
+            var inProcDataCollectorNode = doc.SelectSingleNode("RunSettings/InProcDataCollectionRunSettings/InProcDataCollectors");
+            InProcDataCollectorsAvailable = inProcDataCollectorNode != null && inProcDataCollectorNode.SelectNodes("InProcDataCollector").Count > 0;
+            if (InProcDataCollectorsAvailable)
```

Comment:
If we remove the two settings, that leaves NumberOfTestWorkers, which may have been set in conflict with InProcDataCollectorsAvailable. I wouldn't override that conflict here but would lean toward overriding it in the package settings with a warning message.